### ISSUE TITLE
Can linux run 2024 Solidworks

### ISSUE THREAD

**Issue by helaeazy** [cryinkfly/SOLIDWORKS-for-Linux#72]
I have a Surface Laptop Go 3. I installed everything that was needed, but in the end, it shows a black screen (see the attachment).
![Screenshot from 2024-10-13 15-23-04](https://github.com/user-attachments/assets/bcb15897-5198-41ff-9e55-8712e74ba8f9)

P.S. I bought a version of SolidWorks when I had Windows.

**Comment by mike-lloyd03** [comment]
I'm having the same issue. Blank window in front of the installer window. Nothing else I can do at this point.

**Comment by helaeazy** [comment]
The best thing to do is install windows and linux on same device just switch when needed for solidoworks. You cant install it on linux myb in future

**Comment by mike-lloyd03** [comment]
Yeah that's what I'm currently doing but it's a pain to reboot just to grab a quick dimension or a screenshot or something.
